about.html

```diff
@@ -0,0 +1,156 @@
+                        </p>
+                    </div>
+                    <div class="col-lg-8 col-md-6 text-center">
+
+                        <h2>Please contact us per email for any further questions about Developer Days 2021 !</h2>
+                        <h2><b>contact@devdays.org</b></h2>
+
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+
+    <section class="stn-about">
+        <div class="container">
+            <div class="row justify-content-center">
+                <div class="col-lg-8">
+                    <div class="main-title">
+                        <h1 class="title">Developer Days Côte d'Ivoire 2021</h1>
+                        <div class="title-line"></div>
+                    </div>
+                    <p>
+                        The logo of CC Global Summit 2015 was decided through the logo competition from 8 June to 7 July. After online public voting and Selection Committee's judging process, the logo from Naresh Agrawal won the prize. After consultation with Mr. Agrawal we
+                        optimized the prize-winning-logo for this year's summit as following:
+                    </p>
+
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="stn-about">
+        <div class="main-title">
+            <h1 class="title">See the past Developer Days Côte d'Ivoire</h1>
+            <div class="title-line"></div>
+        </div>
+        <div class="row justify-content-center">
+            <div class="col-lg-4 col-md-6">
+                <div class="past-event event1">
+                    <h4>2019</h4>
+                    <p>Developer Days Côte d'Ivoire</p>
+                </div>
+            </div>
+
+            <div class="col-lg-4 col-md-6">
+                <div class="past-event event2">
+                    <h4>2020</h4>
+                    <p>Developer Days Côte d'Ivoire</p>
+                </div>
+            </div>
+
+        </div>
+    </section>
+    <footer class="section-footer abt-dark">
+        <div class="container">
+            <div class="row justify-content-center">
+                <div class="col-lg-2 col-md-6">
+                    <div class="logo">
+                        <a href="index.html"><i class="fa fa-terminal" aria-hidden="true"></i> <span>Dev Days</span></a>
+                    </div>
+                </div>
+                <div class="col-lg-10 col-md-6">
+                    <div class="footer-copy-right text-center">
+
+                        <h4>Copyright © 2021 All rights reserved | This website is made with <i class="fa fa-heart color-danger" aria-hidden="true"></i> with <a href="https://getbootstrap.com/">bootstrap .</a></h4>
+
+                    </div>
+                </div>
+            </div>
+        </div>
+    </footer>
+
+
+
+
```

Comment:
- [ ] OPTIONAL
To enhance a cleaner code kindly endeavor to remove **all** extra lines which are serving no purpose in your code.